app/main/presenters/search_presenters.py

```diff
@@ -193,7 +200,11 @@ def build_lots_and_categories_link_tree(framework, lots, category_filter_group,
 
     # Links in the tree should preserve all the filters, except those relating to this tree (i.e. lot
     # and category).
-    search_link_builder = Href(url_for('.search_services'))
+    if framework['framework'] == 'g-cloud':
```

Comment:
Maybe we should pass the 'link builder' in - so that we can avoid having this test altogether?

Reply:
Nice. yes. done.